tests/test_core.py

```diff
@@ -844,16 +836,41 @@ def test_no_duplicate_protocols(self):
             class DuplicateProtocol(NSObject, protocols=[NSObjectProtocol, NSObjectProtocol]):
                 pass
 
+    def test_class_ivars(self):
+        """An Objective-C class can have instance variables."""
+
+        class Ivars(NSObject):
+            object = objc_ivar(objc_id)
+            int = objc_ivar(c_int)
+            rect = objc_ivar(NSRect)
+
+        ivars = Ivars.alloc().init()
+
+        set_ivar(ivars, 'object', at('foo').ptr)
+        set_ivar(ivars, 'int', c_int(12345))
+        set_ivar(ivars, 'rect', NSMakeRect(12, 34, 56, 78))
```

Comment:
Is this a deliberately explicit test of`set_ivar`, rather than using "Pythonic" assignment to `ivars.object` etc? I *think* it is; but regardless, for the sake of clarity, a comment explaining what exactly is being tested would be helpful.

Reply:
Python attribute syntax doesn't work with ivars, only with properties, which are tested separately. The only way to get and set ivars is with the `get_ivar` and `set_ivar` functions. (This behavior is not new by the way, attribute syntax has never worked with ivars before, as far as I know.)

Reply:
ACK - makes sense. At some point in the future, we should probably document when you should use an `objc_property`, when you should use an `objc_ivar`, and when you should use a Python property.